cli.js

```diff
@@ -1,30 +1,41 @@
 #!/usr/bin/env node
 'use strict';
 
-var path = require('path');
 var childProcess = require('child_process');
+var meow = require('meow');
 var globby = require('globby');
 var inquirer = require('inquirer');
 var assign = require('lodash.assign');
+var npmRunPath = require('npm-run-path');
 var utils = require('./cli-utils');
 var codemods = require('./codemods.json');
 
 function runScripts(scripts, files) {
-	var binPath = require.resolve('jscodeshift/bin/jscodeshift.sh');
+	// var binPath = require.resolve('jscodeshift/bin/jscodeshift.sh');
 	var spawnOptions = {
-		env: assign({}, process.env, {PATH: path.resolve('node_modules/.bin') + ':' + process.env.PATH}),
+		env: assign({}, process.env, {PATH: npmRunPath()}),
 		stdio: 'inherit'
 	};
 
 	var result;
 	scripts.forEach(function (script) {
-		result = childProcess.spawnSync(binPath, ['-t', script].concat(files.split(' ')), spawnOptions);
+		result = childProcess.spawnSync('jscodeshift', ['-t', script].concat(files), spawnOptions);
 		if (result.error) {
 			throw result.error;
 		}
 	});
 }
 
+meow(`
+		Usage
+			$ ava-codemods
+
+		Ensure you have a backup of your tests or commit the latest changes before running this.
```

Comment:
I don't think it should be here as most won't read the `--help`. Either readme or in the Inquirer dialog.